php/driftphp/config.yaml

```diff
@@ -10,4 +10,4 @@ php_mod:
   - posix
   - sockets
 
-command: php vendor/bin/server run 0.0.0.0:3000 --env=prod --quiet
\ No newline at end of file
+command: php vendor/bin/server run 0.0.0.0:3000 --env=prod --quiet --workers=-1
```

Comment:
is it more more interesting to spread across all CPU ? instead of using 1 worker

please note that I'm not sure that is useful, since I don't know how reactphp react 😛